users/utils.py

```diff
@@ -0,0 +1,25 @@
+import jwt
+import json
+import requests
+
+from django.http import JsonResponse
+from django.conf import settings
+from users.models import User
+
+
+def login_decorator(func):
+    def wrapper(self, request, *args, **kwargs):
+        try:
+            access_token  = request.headers['Authorization']
+            payload       = jwt.decode(access_token, settings.SECRET_KEY, algorithms='HS256')
+            user          = User.objects.get(id=payload['id'])
+            request.user  = user.id
```

Comment:
id를 넘기기보다는 객체를 넘겨주세요.
```suggestion
            request.user  = user
```